src/SourceBuild/content/eng/tools/init-build.proj

```diff
@@ -113,15 +113,15 @@
   </Target>
 
   <Target Name="GenerateRootFs"
-          Condition="'$(BuildOS)' != 'windows'">
+          Condition=" '$(BuildOS)' != 'windows' AND '$(CrossBuild)' == 'true' AND $(ROOTFS_DIR) == '' ">
```

Comment:
The usual style nits: Here and in most other repos (and also largely in dotnet/runtime) we don't add a whitespace around a Condition statement. Also, we use lower case operators.

```suggestion
          Condition="'$(BuildOS)' != 'windows' and '$(CrossBuild)' == 'true' and '$(ROOTFS_DIR)' == ''">
```